rust-toolchain.toml

```diff
@@ -1,3 +1,4 @@
 [toolchain]
-channel = "nightly-2022-07-24"
+channel = "nightly-2022-11-29"
```

Comment:
Im not in favor of using a newer version than in the last Polkadot release. But we could try until we run into some weird compilation errors.